modApiServer/test/org/aion/api/server/TxRecptLgTest.java

```diff
@@ -0,0 +1,162 @@
+
+        StandaloneBlockchain.Bundle bundle = (new StandaloneBlockchain.Builder())
+                .withValidatorConfiguration("simple")
+                .withDefaultAccounts()
+                .build();
+        StandaloneBlockchain bc = bundle.bc;
+        ECKey deployerAccount = bundle.privateKeys.get(0);
+
+        //======================
+        // DEPLOY contract A & B
+        //======================
+        BigInteger nonce = BigInteger.ZERO;
+        AionTransaction tx1 = new AionTransaction(
+                nonce.toByteArray(),
+                null,
+                new byte[0],
+                ByteUtil.hexStringToBytes(contractA),
+                1_000_000L,
+                1L
+        );
+        tx1.sign(deployerAccount);
+
+        nonce = nonce.add(BigInteger.ONE);
+        AionTransaction tx2 = new AionTransaction(
+                nonce.toByteArray(),
+                null,
+                new byte[0],
+                ByteUtil.hexStringToBytes(contractB),
+                1_000_000L,
+                1L
+        );
+        tx2.sign(deployerAccount);
+
+        BlockContext context = bc.createNewBlockContext(bc.getBestBlock(), List.of(tx1, tx2), false);
+        ImportResult result = bc.tryToConnect(context.block);
+        assertEquals(result, ImportResult.IMPORTED_BEST);
+
+        Address addressA = tx1.getContractAddress();
+        System.out.println("contract A = " + addressA);
+        Address addressB = tx2.getContractAddress();
+        System.out.println("contract B = " + addressB);
+        Thread.sleep(1000);
+
+        //======================
+        // CALL function A.AA
+        //======================
+        nonce = nonce.add(BigInteger.ONE);
+        AionTransaction tx3 = new AionTransaction(
+                nonce.toByteArray(),
+                addressA,
+                new byte[0],
+                ByteUtil.merge(ByteUtil.hexStringToBytes("0x9dacb3a3"), addressB.toBytes(), new DataWord(80_000).getData()),
+                1_000_000L,
+                1L
+        );
+        tx3.sign(deployerAccount);
+
+        context = bc.createNewBlockContext(bc.getBestBlock(), List.of(tx3), false);
+        result = bc.tryToConnect(context.block);
+        assertEquals(result, ImportResult.IMPORTED_BEST);
+
+        AionTxInfo info = bc.getTransactionInfo(tx3.getHash());
+        AionTxReceipt receipt = info.getReceipt();
+        System.out.println(receipt);
+        Assert.assertEquals(4, receipt.getLogInfoList().size());
```

Comment:
I think you can just say assertEquals, without the "Assert."

Reply:
:+1: